misk/src/main/kotlin/misk/client/ClientInvocationHandler.kt

```diff
@@ -64,11 +72,34 @@ internal class ClientInvocationHandler(
 
     if (tracer != null) retrofitBuilder.callFactory(TracingCallFactory(actionSpecificClient, tracer))
 
+    val mediaTypes = getEndpointMediaTypes(methodName)
+    if (mediaTypes.contains(MediaTypes.APPLICATION_PROTOBUF)) {
+      retrofitBuilder.addConverterFactory(WireConverterFactory.create())
+    }
+    // Always add JSON as default. Ensure it's added last so that other converters get a chance since
```

Comment:
what does it mean to allow multiple converters? shouldn't only a single converter be installed?

Reply:
Previous comment should clarify this.